db/migrate/20231205095734_add_stock_rna_plate_to_working_dilution_parent_purposes.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# This migration adds the 'Stock RNA Plate' plate purpose to the parent plate
+# purposes of the 'Working dilution' plate creator.
+#
+# If the 'Stock RNA Plate' plate purpose does not exist in the database, it is
+# created first.
+#
+# It is assumed that the 'Working dilution' plate creator exists and its plate
+# purposes contain 'Working Dilution' purpose but its parent plate purposes
+# does not contain the 'Stock RNA Plate' purpose.
+#
+# If the migration is rolled back, the 'Stock RNA Plate' purpose is removed from
+# the parent plate purposes of the 'Working dilution' plate creator. However,
+# the 'Stock RNA Plate' plate purpose is kept in the database.
+#
+class AddStockRnaPlateToWorkingDilutionParentPurposes < ActiveRecord::Migration[6.0]
```

Comment:
It was not possible to the record loader for this because it does not allow updates.

I have removed the migration and seeds now. I added a rake task. It is just for adding a record but the benefit is having a test going together.